src/core/klsh.js

```
@@ -7,42 +7,60 @@ function clone(obj) {
+    const argObj = literal_to_string(p, env);
+    stderr += argObj.stderr;
+    args.push(argObj.text);
+  }
+  const builtin = klsh[name];
+  if (builtin && typeof builtin.main === 'function') {
+    try {
+      const res = builtin.main({ args, stdin: input, env });
+      stderr += res.stderr;
+      return { stdout: res.stdout, stderr, env: clone(res.env) };
+    } catch (err) {
+      stderr += (err.message || 'Error') + '\n';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)